help/destinations/catalog/advertising/liveramp-distribution.md

```diff
@@ -44,13 +42,13 @@ To help you better understand how and when you should use the [!DNL LiveRamp - D
 
 The marketing team of a sports apparel retailer used the [LiveRamp - Onboarding](liveramp-onboarding.md) connection to send audiences from Experience Platform to their LiveRamp account.
 
-Through the [!DNL LiveRamp - Distribution] connection they can now trigger the activation of the onboarded audiences to the destinations mentioned at the top of this page, so that they can target users on mobile, open web, social, and [!DNL CTV] platforms.
+Through the [!DNL LiveRamp - Distribution] connection they can now trigger the activation of the onboarded audiences to the destinations mentioned at the top of this page directly within Adobe Real Time CDP, so that they can target users on mobile, open web, social, and [!DNL CTV] platforms.
```

Comment:
The correct branding is Adobe Real-Time CDP. Please make sure this is the form used everywhere you reference it.